types/configs/i18n-dictionary.d.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Structure of the i18n dictionary
+ */
+export interface I18nDictionary {
+  /**
+   * Translation of internal UI components of the editor.js core
+   */
+  ui: Dictionary;
```

Comment:
I think all fields should be optional (if we should be able to translate some parts and don't translate other) or required (if a dictionary is required to be applied to all parts)